src/api/SFA.DAS.Provider.Events.Infrastructure/Data/DcfsPaymentRepository.cs

```diff
@@ -77,30 +78,46 @@ public class DcfsPaymentRepository : DcfsRepository, IPaymentRepository
                     childCollection: p => p.PaymentsDueEarningEntities
                 ).ConfigureAwait(false);
 
-                var pagedResults =  PageResults(result, page, pageSize);
+                var pagedResults = PageResults(result, page, pageSize);
                 return pagedResults;
             }
         }
 
+        public async Task<PaymentStatistics> GetStatistics()
+        {
+            using (var connection = await GetOpenConnection().ConfigureAwait(false))
+            {
+                var sql = "    SELECT count(1) as TotalPayments" +
+                          "    FROM Payments.Payments p " +
+                          "    INNER JOIN PaymentsDue.RequiredPayments rp ON p.RequiredPaymentId = rp.Id ";

```

Comment:
Is the join required to get the count?

Reply:
as per previous conversation, stats updated to include additional stat as numbers dont match but should!